ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -312,7 +384,7 @@ public void onInstanceConfigChange(List<InstanceConfig> configs, NotificationCon
             initializationException.compareAndSet(null, e);
           }
           instanceConfigInitialized.set(true);
-        } else {
+        } else if (!clusterMapConfig.clusterMapEnablePartitionOverride) {
           updateSealedStateOfReplicas(configs);
```

Comment:
your other PR made a change to this function (to account for stopped replicas too) - does this change in the `if` still hold? (I think it should)

Reply:
hmm, i think here might be a problem. If `clusterMapEnablePartitionOverride` is enabled, looks like we cannot dynamically update the `STOPPED` state of replica. What do you think? (I didn't consider this case in that PR)